tests/unit/matrix/test_measure.py

```diff
@@ -115,6 +119,40 @@ def it_provides_access_to_the_cube_measures_to_help(
         CubeMeasures_.assert_called_once_with(cube_, dimensions_, 42)
         assert cube_measures is cube_measures_
 
+    @pytest.mark.parametrize(
+        "measure_prop_name, MeasureCls",
+        (
+            ("pairwise_t_stats", _PairwiseSigTstats),
+            ("pairwise_p_vals", _PairwiseSigPvals),
+            ("pairwise_significance_means_p_vals", _PairwiseMeansSigPVals),
+            ("pairwise_significance_means_t_stats", _PairwiseMeansSigTStats),
+        ),
+    )
+    def it_provides_access_to_pairwise_significance_measure_objects(
+        self,
+        request,
+        dimensions_,
+        _cube_measures_prop_,
+        cube_measures_,
+        measure_prop_name,
+        MeasureCls,
+    ):
+        measure_ = instance_mock(request, MeasureCls)
+        MeasureCls_ = class_mock(
+            request,
+            "cr.cube.matrix.measure.%s" % MeasureCls.__name__,
+            return_value=measure_,
+        )
+        _cube_measures_prop_.return_value = cube_measures_
+        measures = SecondOrderMeasures(None, dimensions_, None)
+        measure = getattr(measures, measure_prop_name)
+        col_idx = 0
+
+        assert measure(col_idx) is measure_
```

Comment:
This is a bit convoluted. I had a hard time figuring out what is what and what you're trying to assert. I'd prefer if instead of `measure` you'd use something more similar to `MeasureCls`, or something more appropriate. The main issue is that I don't understand what the `measure(col_idx)` call is supposed to be doing, because I don't know what `measure` is supposed to be. Also, while you're at it, please separate out that call on its own line, and do the assertions afterwards.